en/rest/hooks.mdown

```diff
@@ -0,0 +1,670 @@
+# Hooks
+
+You can create, update, list or delete all your cloud code webhooks via the Hooks API,
+in addition to being able to do so through the parse website.
+
+Hooks API requires that the user provides `Application-Id` and `Master-Key` in the request headers.
+
+There are two kinds of cloud code webhooks: cloud function hooks and object trigger hooks.
+
+Cloud functions are functions that run in the cloud and allow you to build functions
+common to all platforms. They can be changed without redeploying your app.
```

Comment:
"redeploying" is confusing.  Does it mean running "parse deploy" on the CLI or shipping a new release to the app store?  Is there a better term?